src/rastervision/common/models/custom_layers.py

```diff
@@ -0,0 +1,67 @@
+from keras import initializers
```

Comment:
You should add comments at the top of these three files noting where they were modified from as well as the license of the repo they come from.